frontend/src/app/page.tsx

```diff
@@ -436,7 +440,7 @@ const HomePage: React.FC = () => {
             )}
           </>
         )}
-  
```

Comment:
I guess your IDE does this formatting but nevertheless I suggest to inspect your staged changes to avoid PR commits with a bunch of random line deletion/additions. merging it like this tho, just for future reference :D thanks for taking the time!

Reply:
Sure, thanks for the tip!
And it was fun to work on it :)))
I'm happy the PR is useful for the project!

Reply:
Be sure to join the discord https://discord.com/invite/R2yKBd42